Tasks/HelmDeployV0/task.loc.json

```diff
@@ -14,7 +14,7 @@
   "version": {
     "Major": 0,
     "Minor": 241,
```

Comment:
This won't be delivered within 241 sprint, so you need to use 242.0 major and minor versions here